design/resources.go

```diff
@@ -612,82 +612,110 @@ var _ = a.Resource("work-item-link-type", func() {
 
 var _ = a.Resource("work-item-link", func() {
 	a.BasePath("/workitemlinks")
+	a.Action("show", showWorkItemLink)
+	a.Action("list", listWorkItemLinks)
+	a.Action("create", createWorkItemLink)
+	a.Action("delete", deleteWorkItemLink)
+	a.Action("update", updateWorkItemLink)
+})
 
-	a.Action("show", func() {
-		a.Routing(
-			a.GET("/:id"),
-		)
-		a.Description("Retrieve work item link (as JSONAPI) for the given ID.")
-		a.Params(func() {
-			a.Param("id", d.String, "ID of the work item link")
-		})
-		a.Response(d.OK, func() {
-			a.Media(WorkItemLink)
-		})
-		a.Response(d.BadRequest, JSONAPIErrors)
-		a.Response(d.InternalServerError, JSONAPIErrors)
-		a.Response(d.NotFound, JSONAPIErrors)
-	})
-
+var _ = a.Resource("work-item-relationships-links", func() {
+	a.BasePath("/relationships/links")
+	a.Parent("workitem")
+	a.Action("show", showWorkItemLink)
```

Comment:
Agreed. Thank you for sharing your thoughts. @aslakknutsen I'd like to remove the CUD operations from the `/api/workitems/:id/relationships/links/:linkid` endpoint. Such operations have to made on the actual resource endpoint `/api/work-item-links/:linkid`.